hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -646,7 +646,8 @@ public Response head(
 
     ResponseBuilder response = Response.ok().status(HttpStatus.SC_OK)
         .header("Content-Length", key.getDataSize())
-        .header("Content-Type", "binary/octet-stream");
+        .header("Content-Type", "binary/octet-stream")
+        .header("x-amz-storage-class", "STANDARD");
```

Comment:
Currently, there is a conversion relationship between Ozone's `StorageType` and `ReplicationFactor`, please refer to `S3Utils#resolveS3ClientSideReplicationConfig`.
You can get a key's `x-amz-storage-class` based on `S3StorageType`